### DISCUSSION TITLE
4 things

### DISCUSSION THREAD

**Discussion by theatomicboy81** [UkrainianBanderasCat/Digital-Logic-Sim2#28] [General]
how do you rotate chips
add more instructions
add the ability to import chips to other saves
oh and performance

**Comment by AlphaCodingPilot** [comment]
You can rotate all selected chips clockwise with E and counterclockwise with Q.

**Comment by UkrainianBanderasCat** [comment]
Hmm...  Importing chips is a perfect idea
But there needs to be one thing... Import with chip other chips which it uses...

**Reply by UkrainianBanderasCat** [reply]
theatomicboy81, do you want to write new instructions?

**Reply by theatomicboy81** [reply]
?, if that means coding sorry but i cant code, I saw this while looking to download the original for a new pc and found this and have been giving ideas, again sorry, tho i have tried unity before and tried learning c# but i couldnt find good tutorials on yt. sorry if that was a long answer

**Reply by firecerne** [reply]
No, you can just write new instructions and I will probably implement it. Just send it here.

**Reply by theatomicboy81** [reply]
Maybe you rc the chip and it says cancel delete transfer to other save then if you click the ladder it opens a box like in the load game thing and then after you click what save you want it shows the chips under it asks you which you need to transfer so you don’t end up with like 30 or gates or something like that.

**Reply by firecerne** [reply]
I was talking about the instructions to help you in the Menu like your second proposition but I'll try to do that.

**Reply by firecerne** [reply]
What if you have two search bars : one for your project's chips and the other for every chip your ever created ?

**Reply by theatomicboy81** [reply]
thats a good idea
and the instructions
"To transfer a chip from a save to another, first right click the chip and select the save you want to transfer, then you select the chips inside the chip you want to transfer."

**Reply by AlphaCodingPilot** [reply]
You could have a button called 'import' in the topbar next to 'create'. When you click on it, it whould show all chips from other projects for you to import.

**Reply by theatomicboy81** [reply]
that would be good as well

**Reply by firecerne** [reply]
> You could have a button called 'import' in the topbar next to 'create'. When you click on it, it whould show all chips from other projects for you to import.

I'll add that

**Reply by firecerne** [reply]
I'm working on it. I think I'm making it overcomplicated but it should work

**Comment by AlphaCodingPilot** [comment]
What do you mean by add more instruction?
